src/demo.js

```diff
@@ -127,6 +130,10 @@ export default class DemoTimeline extends Component {
     let newMode = timelineMode ^ TIMELINE_MODES.RESIZE;
     this.setState({timelineMode: newMode, message: 'Timeline mode change: ' + timelineMode + ' -> ' + newMode});
   }
+  toggleTimestamps() {
+    const {useTimestamps} = this.state;
+    this.setState({useTimestamps: !useTimestamps});
```

Comment:
here we can see what I was saying for the other review. W/o destructuring, we'd have only one line